src/cow-react/modules/limitOrders/pure/RateImpactIndicator/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { AlertTriangle } from 'react-feather'
+import styled from 'styled-components/macro'
+import { MouseoverTooltipContent } from 'components/Tooltip'
+import { LOW_RATE_THRESHOLD_PERCENT } from '@cow/modules/limitOrders/const/trade'
+
+interface RateImpactProps {
+  rateImpact: number
+}
+
+const PercentBox = styled.span<{ isPositive: boolean; isTooLowRate: boolean }>`
+  font-size: 12px;
+  display: inline-flex;
+  align-items: center;
+  margin-left: 5px;
+  color: ${({ isPositive, isTooLowRate }) => (isPositive ? 'green' : isTooLowRate ? 'red' : 'grey')};
+
+  svg {
+    margin-left: 5px;
+  }
+`
+const ImpactTooltip = styled.span`
+  padding: 5px 10px;
+`
+
+export function RateImpactIndicator({ rateImpact }: RateImpactProps) {
+  const isPositive = rateImpact > 0
+  const hugePositiveImpact = rateImpact > 999
```

Comment:
Where does this 999 come from?

Reply:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/7122625/200784897-c3e11f87-02a4-4f48-86ce-3f91d231d3dd.png">

It's just to avoid cases like on the picture above.

Reply:
So, if rate impact is more than 999% we display it as `>999%`

Reply:
Moved it to const